lib/src/models/invite/invite.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:nyxx/src/models/user/user.dart';
+import 'package:nyxx/src/models/invite/stage_instance.dart';
+
+/// An invite to a [Guild] or [Channel].
+/// If the invite is to a [Channel], this will be a [ChannelType.groupDm] channel.
+class Invite {
```

Comment:
```suggestion
import 'package:nyxx/src/utils/to_string_helper/to_string_helper.dart';

/// An invite to a [Guild] or [Channel].
/// If the invite is to a [Channel], this will be a [ChannelType.groupDm] channel.
class Invite with ToStringHelper {
```

Since this isn't a SnowflakeEntity, it should have `ToStringHelper` mixed in to get a nice output when debugging. `InviteMetadata` will get it by inheritance.